sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/TestSuiteBase.java

```diff
@@ -1125,4 +1127,88 @@ protected int expectedNumberOfPages(int totalExpectedResult, int maxPageSize) {
     public static CosmosClientBuilder copyCosmosClientBuilder(CosmosClientBuilder builder) {
         return CosmosBridgeInternal.cloneCosmosClientBuilder(builder);
     }
+
+
+    public enum City {
```

Comment:
please move the test POJO types to standalone files outside of TestSuiteBase. Same as Pet class, etc.

Reply:
Sure will move